.gitignore

```diff
@@ -1,3 +1,6 @@
+# qgis-boilerplate
```

Comment:
Add comment: boiler plate build qgis environment and starts application

Reply:
are you suggesting we change this? In order to run install outside of QGIS, the boilerplate is required (to get at Q profile mgr + QSettings), so I figured we could drop it in the folder on P: but wouldn't want to upload it to GitHub (though maybe it's OK to upload, it doesn't show too much of anything).

I modified the install script to work from Python console in Q for folks without a boilerplate setup